js/settingsHelpers.js

```diff
@@ -733,22 +733,22 @@ export const validateAltContactInformation = async (altContactMobilePhoneComplet
     hasError = true;
   }
 
-  // if (!hasError) {
-  //   const emailValidation = await emailAddressValidation({
-  //     emails: {
-  //       altContactEmail: altContactEmail || undefined,
-  //     },
-  //   });
-
-  //   const upEmailValidationAnalysis = emailValidationAnalysis(emailValidation.altContactEmail)
-  //   if (upEmailValidationAnalysis === emailValidationStatus.WARNING) riskyEmails.push(altContactEmail)
-  //   if (upEmailValidationAnalysis === emailValidationStatus.INVALID) {
-  //     errorMessage('newAltContactEmail', translateText('settingsHelpers.emailInvalid'), focus);
-  //     if (focus) document.getElementById('newAltContactEmail').focus();
-  //     focus = false;
-  //     hasError = true;
-  //   }
-  // }
+  if (!hasError) {
+    const emailValidation = await emailAddressValidation({
+      emails: {
+        altContactEmail: altContactEmail || undefined,
```

Comment:
Since `altContactEmail` should pass email format check if code run reaches here, it should have valid value.
```suggestion
        altContactEmail
```